cognite/client/data_classes/data_modeling/views.py

```diff
@@ -386,17 +387,19 @@ class SingleHopConnectionDefinition(ConnectionDefinition):
     description: str | None = None
     edge_source: ViewId | None = None
     direction: Literal["outwards", "inwards"] = "outwards"
+    connection_type: Literal["multi_edge_connection"] = "multi_edge_connection"
```

Comment:
Hmm. Is it really necessary to snake case the enum?

Reply:
Is just that we do it as far as I know everywhere else in the SDK.